src/CMakeLists.txt

```diff
@@ -782,8 +782,11 @@ if (NOT WIN32)
   endif() # TRITON_ENABLE_GPU
 endif() # NOT WIN32
 
-# tritonfrontend python package
-add_subdirectory(python)
+# DLIS-7292: Extend tritonfrontend to build for Windows
+if (NOT WIN32)
```

Comment:
Were other parts broken other than the CURL/tracing/otel stuff?

Reply:
IIRC, the build would fail if individual parts like TRACING were disabled (which we have a follow-up ticket for) - so it would make sense to me if skipping tracing portions due to windows would cause the whole thing to fail.